content/pt-br/docs/concepts/windows/intro.md

```diff
@@ -0,0 +1,281 @@
+
+## Container de pausa
+
+Em um Pod Kubernetes, um container de infraestrutura ou "pausa" é criado primeiro para hospedar o container. No Linux, os cgroups e namespaces que compõem um pod precisam de um processo para manter sua existência contínua; o processo de pausa fornece isso. Containers que pertencem ao mesmo pod, incluindo infraestrutura e containers de trabalho, compartilham um endpoint de rede comum (mesmo endereço IPv4 e/ou IPv6, mesmos espaços de porta de rede). O Kubernetes usa containers de pausa para permitir que containers de trabalho falhem ou reiniciem sem perder qualquer configuração de rede.
+
+O Kubernetes mantém uma imagem multi-arquitetura que inclui suporte para Windows. Para o Kubernetes v{{< skew currentPatchVersion >}} a imagem de pausa recomendada é `registry.k8s.io/pause:3.6`. O [código fonte](https://github.com/kubernetes/kubernetes/tree/master/build/pause) está disponível no GitHub.
+
+A Microsoft mantém uma imagem multi-arquitetura diferente, com suporte para Windows amd64 e Linux, que você pode encontrar como `mcr.microsoft.com/oss/kubernetes/pause:3.6`. Esta imagem é construída a partir do mesmo código fonte que a imagem mantida pelo Kubernetes, mas todos os binários do Windows são [assinados pelo Authenticode](https://docs.microsoft.com/pt-br/windows-hardware/drivers/install/authenticode) pela Microsoft. O projeto Kubernetes recomenda usar a imagem mantida pela Microsoft se você estiver implantando em um ambiente de produção ou similar que exija binários assinados.
+
+## Runtimes de container {#container-runtime}
+
+Você precisa instalar um {{< glossary_tooltip text="runtime de container" term_id="container-runtime" >}} em cada nó do cluster para que os Pods possam ser executados lá.
+
+Os seguintes runtimes de container funcionam com Windows:
+
+{{% thirdparty-content %}}
+
+### ContainerD
+
+{{< feature-state for_k8s_version="v1.20" state="stable" >}}
+
+Você pode usar {{< glossary_tooltip term_id="containerd" text="ContainerD" >}} 1.4.0+ como o runtime de container para nós Kubernetes que executam Windows.
+
+Aprenda como [instalar o ContainerD em um nó Windows](/docs/setup/production-environment/container-runtimes/#containerd).
+
+{{< note >}}
+Há uma [limitação conhecida](/pt-br/docs/tasks/configure-pod-container/configure-gmsa/s) ao usar GMSA com containerd para acessar compartilhamentos de rede do Windows, o que requer um patch no kernel.
+{{< /note >}}
+
+### Mirantis Container Runtime {#mcr}
+
+[Mirantis Container Runtime](https://docs.mirantis.com/mcr/20.10/overview.html) (MCR) está disponível como um runtime de container para todas as versões do Windows Server 2019 e posteriores.
+
+Veja [Instalar MCR em servidores Windows](https://docs.mirantis.com/mcr/20.10/install/mcr-windows.html) para mais informações.
+
+## Compatibilidade de versão do Windows OS {#windows-os-versãos-suporte}
```

Comment:
Esta âncora precisa ser mantida no original para que o link do início da página funcione.

```suggestion
## Compatibilidade de versão do Windows OS {#windows-os-version-support}
```